cmd/src/cmd/stream/delete.rs

```diff
@@ -30,7 +30,7 @@ impl CliCommand for StreamDelete {
                 stream_id: Identifier::numeric(self.id).expect("Expected numeric identifier"),
             })
             .await
-            .with_context(|| format!("Problem deleting stream (id: {})", self.id))?;
+            .with_context(|| format!("Problem deleting stream with id: {}", self.id))?;
```

Comment:
Maybe we could use the uppercase `ID` everywhere? Just a random thought :)

Reply:
Ok, good idea. I'll include it in this PR